start_bundle_coupled.R

```diff
@@ -240,20 +245,24 @@ for(scen in common){
 
   #cfg$logoption  <- 2  # Have the log output written in a file (not on the screen)
 
-  # Add non-gms-switches manually
-  if( "regionmapping" %in% names(settings_remind)){
-    cfg_rem$regionmapping <- settings_remind[scen,"regionmapping"]
+  # Edit remind main model file, region settings and input data revision based on scenarios table, if cell non-empty
+  for (switchname in intersect(c("model", "regionmapping", "inputRevision"), names(settings_remind))) {
+    if ( ! is.na(settings_remind[scen, switchname] )) {
+      cfg_rem[[switchname]] <- settings_remind[scen, switchname]
+    }
   }
 
-  # Edit default.cfg settings according to the SSP scenarios only for elements in 'scenarios' that exist in the cfg
-  for (switchname in intersect(names(cfg_rem$gms),names(settings_remind))){
-    cfg_rem$gms[[switchname]] <- settings_remind[scen,switchname]
+  # Edit switches in default.cfg based on scenarios table, if cell non-empty
+  for (switchname in intersect(names(cfg_rem$gms), names(settings_remind))) {
+    if ( ! is.na(settings_remind[scen, switchname] )) {
+      cfg_rem$gms[[switchname]] <- settings_remind[scen, switchname]
+    }
   }
 
   # If provided replace gdx paths given in scenario_config with paths given in scenario_config_coupled
   if (!is.na(scenarios_coupled[scen, "path_gdx_bau"])) {
-	  settings_remind[scen, "path_gdx_bau"] <- scenarios_coupled[scen, "path_gdx_bau"]
-	  cat("Replacing gdx_bau information with those specified in\n  ",path_settings_coupled,"\n  ",settings_remind[scen, "path_gdx_bau"],"\n")
+          settings_remind[scen, "path_gdx_bau"] <- scenarios_coupled[scen, "path_gdx_bau"]
```

Comment:
could this cause problems in coupled runs trying to find the last coupled iteration?

Reply:
As far as I can see, the only thing I changed here was tabs to spaces